.github/workflows/pages.yml

```diff
@@ -37,6 +37,7 @@ jobs:
           cache-version: 0 # Increment this number if you need to re-download cached gems
```

Comment:
Commenting here because the stuff above is not part of the diff, but it's about the `concurrency` block that it's at the `job` level. Move it under the `deploy` job.

Reply:
Done